src/app/client/src/app/plugins/profile/components/create-user/create-user.component.html

```diff
@@ -1,62 +1,68 @@
+  <div [ngClass]="layoutConfiguration ? 'sb-back-actionbar' : 'sb-bg-white cc-player__btn-back'"
+    class="relative position mt-0">
+    <div class="ui container py-0 px-0 d-flex flex-ai-center">
+      <button type="button" (click)="goBack()"
+        [ngClass]="layoutConfiguration ? 'sb-btn-primary sb-btn-round' : 'sb-btn-link sb-btn-link-primary sb-left-icon-btn px-0'"
+        class="sb-btn sb-btn-normal" tabindex="0" attr.aria-label="{{resourceService?.frmelmnts?.btn?.back}}">
         <i class="icon-svg icon-svg--xxs icon-back mr-4"><svg class="icon icon-svg--primary">
             <use xlink:href="assets/images/sprite.svg#arrow-long-left"></use>
           </svg></i>
-       <span>{{resourceService?.frmelmnts?.btn?.back}}</span>
+        <span>{{resourceService?.frmelmnts?.btn?.back}}</span>
       </button>
-        <div class="content-header__content w-100 ml-16">
-          <div class="d-flex flex-dc flex-basis-1 mr-32 min-w-0 content-header__content__title">
-            <div class="content-header__title font-weight-bold ellipsis text-left"  tabindex="0" role="heading" aria-level="2">{{resourceService?.frmelmnts?.lbl?.createUser}}</div>
-          </div>
+      <div class="content-header__content w-100 ml-16">
+        <div class="d-flex flex-dc flex-basis-1 mr-32 min-w-0 content-header__content__title">
+          <div class="content-header__title font-weight-bold ellipsis text-left" tabindex="0" role="heading"
+            aria-level="2">{{resourceService?.frmelmnts?.lbl?.createUser}}</div>
         </div>
       </div>
     </div>
-
-    <div [ngClass]="layoutConfiguration ? 'sbt-inside-page-container' : ''">
-  <div class="ui container py-16" *ngIf="showLoader">
-    <app-loader></app-loader>
   </div>
-  <div class="user-profile-box p-0" *ngIf="!showLoader">
-    <div class="d-flex flex-jc-space-between py-16 px-16">
-      <div role="heading" aria-level="1" class="user-profile-box__title">{{resourceService?.frmelmnts?.lbl?.createUser}}</div>
+
+  <div [ngClass]="layoutConfiguration ? 'sbt-inside-page-container' : ''">
+    <div class="ui container py-16" *ngIf="showLoader">
+      <app-loader></app-loader>
     </div>
-    <div class="twelve wide column p-0 mx-16 mb-8 text-center"><img src="../../assets/images/profile-icon.svg" alt="profile-icon"
-        width="88px"></div>
-    <form class="sb-form mb-16 px-16" [formGroup]="userDetailsForm">
-      <div class="ui stackable grid m-0" *ngFor="let field of formData">
-        <div class="twelve wide column p-0" *ngIf="field.inputType==='input' && field.visible===true">
-          <div class="d-flex fields-selection p-0">
-            <div class="sb-field field-focus mb-8 w-100">
-              <input
-                [ngClass]="{'orange-border': userDetailsForm.controls[field.code].touched && userDetailsForm.controls[field.code].errors}"
-                class="sb-form-control" formControlName="{{field.code}}" aria-label="Enter Name" type="text" name="{{field.code}}"
```

Comment:
What is the change in the HTML file. I am able to find what exactly the change in this file. 
Lot of indentation changes in the code.

Reply:
format only and also changes the position of the disabled attribute